config/boards/radxa-e54c.csc

```diff
@@ -1,7 +1,7 @@
 # Rockchip RK3582 SoC
 BOARD_NAME="Radxa E54C"
 BOARDFAMILY="rockchip-rk3588"
-BOARD_MAINTAINER="kamilsaigol"
```

Comment:
@kamilsaigol 

If you want to become a board maintainer, [adjust data here](https://www.armbian.com/update-data/).